templates/vendor/assets/javascripts/spree/frontend/all.js

```diff
@@ -5,5 +5,4 @@
 // the compiled file.
```

Comment:
Can't we move these vendor files into non-spree folders?

Reply:
@kennyadsl I don't think we can because extensions generated with SolidusDevSupport inject their asset links to these vendor paths. Please see the `add_javascripts` and `add_stylesheets` methods in https://github.com/solidusio/solidus_dev_support/blob/6270b0ba37cae69b7a6d4fcac928e0b4e55fa212/lib/solidus_dev_support/templates/extension/lib/generators/%25file_name%25/install/install_generator.rb.tt.

Reply:
If I got it correctly, that is going to work only with the old frontend. When using the new one, ideally no more JS files will be injected this way. @elia what do you think? Too early to remove the support for this?

Reply:
I removed `require spree` and didn't encounter any errors when I tested checkout.

Reply:
TBH, I'm fine with both, on one side very few JS files from extensions will keep working on a front end HTML that deviates conspicuously from the original solidus_frontend, on the other we could try to support the installation process and then let people to fix their app by themselves.